src/Servers/IIS/test/IISIntegration.FunctionalTests/UpgradeFeatureDetectionTests.cs

```diff
@@ -22,16 +22,6 @@ public UpgradeFeatureDetectionTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
-        public Task UpgradeFeatureDetectionEnabled_OutOfProcess_IISExpress_CoreClr_x64_Portable()
```

Comment:
Just curious. Why is this being removed?

Reply:
It was a feature we didn't add to ANCMV1.